src/contentscript/overlay/root/pages/UnderConstructionInfo/UnderConstructionInfo.module.scss

```diff
@@ -932,25 +777,43 @@
   font-size: 10px;
   font-weight: 400;
   line-height: 100%;
-  color: #fff;
+  color: #fff!important;
```

Comment:
Here and below the use of `!important` is a sign of bad coding practice. It will lead to unexpected behaviour in the future. If you have to use this, then something is wrong with the style rules hierarchy. Refactor it.